crunchy/cli/decompress_cmd.py

```diff
@@ -19,23 +20,35 @@ def decompress():
 @click.command()
 @click.argument("spring-path", type=click.Path(exists=True))
 @click.option(
-    "--first", "-f", type=click.Path(exists=False), help="First read in pair",
+    "--first",
```

Comment:
```suggestion
    "--first",
```
First seems so unspecific maybe `first_read`or even `first_read_file`. I know it is longer, but you have to think a lot less 😄 . I saw in the test that you use first_read sometimes

Reply:
Sure